Makefile

```diff
@@ -220,6 +220,8 @@ else
  # Additional optimization flags for release builds are not included at present;
  # adding them will involve refactoring config.nims in the root of this repo
  NIM_PARAMS += -d:release
+ STATUSGO_MAKE_PARAMS += CGO_CFLAGS="-O3"
```

Comment:
hey @jakubgs could you help pls, where we could put this for mobile?

Reply:
Somewhere here:
https://github.com/status-im/status-mobile/blob/0ae24070b00167676411f847a5892223d8133c00/nix/status-go/mobile/build.nix#L44-L52
It would probably make sense to parametrize it. I'm assuming this is the same thing as the `-gccgoflags` flag.